osquery/database/database.cpp

```diff
@@ -650,6 +651,9 @@ Status upgradeDatabase(int to_version) {
     } else {
       db_version = ret.get();
     }
+  } else {
+    LOG(WARNING) << "Failed to obtain database version. Assume version '"
```

Comment:
Thinking about it more, even WARNING may be too much. Partly because it's pretty common, but also because we don't expect end users to do anything, right? It means osquery is going to migrate things, and it should be transparent?

Which makes it a developer oriented debug message :|